### PR TITLE
replace own systemd logic with camptocamp/systemd

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
       "version_requirement": ">= 1.1.2 < 2.0.0"
     },
     {
-      "name": "puppet-systemd",
+      "name": "camptocamp-systemd",
```
Comment (bastelfreak) [inline metadata.json]: ah damn. Again I confused the repo name on github with the forge name of a module....

Comment (jsok) [reply metadata.json]: Hah no problem, it was a learning exercise for me to discover what the `module_install_helper` was doing internally.

**Review by jsok — APPROVED** [review]
Happy to merge now unless you have more changes in this PR?

**Comment by bastelfreak** [comment]
nope, I'm fine with it. Thanks for the fixes.

**Pull request merged**